notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala

```diff
@@ -53,7 +53,8 @@ object DistCopyJob {
         job.getConfiguration.setInt(ReadLimit, conf.readLimit.toBytes.value.toInt)
         job.getConfiguration.setLong(MultipartUploadThreshold, conf.multipartUploadThreshold.toBytes.value)
         job.getConfiguration.setBoolean(CrossValidate, conf.parameters.crossValidate)
-        job.getConfiguration.setBoolean(S3Endpoint, conf.client.getEndpoint)
+        // Is there another way to get the endpoint?
+        job.getConfiguration.set(S3Endpoint, conf.client.getRegion.toAWSRegion.getServiceEndpoint("s3"))
```

Comment:
Would be interested in seeing what the output of this is, does it match the endpoint strings we use explicitly, the only other way would be to pass the endpoint down separately to the client.